.freight.yml

```diff
@@ -32,6 +32,8 @@ steps:
     name: events-subscriptions-consumer
   - image: us.gcr.io/sentryio/snuba:{sha}
     name: transactions-subscriptions-consumer
+  - image: us.gcr.io/sentryio/snuba:{sha}
+    name: sessions-subscriptions-consumer
```

Comment:
Please do not do this yet. 
Technically right now we do not even follow this anymore to deploy. THat would be a change in the Kubernetes config. But starting the consumer in prod would be the last step after the topic is created in prod and we tested that the subscription consumer works.

Reply:
Sure I will take it out